app/models/tasks/evidence_submission_window_task.rb

```diff
@@ -11,6 +11,7 @@ class EvidenceSubmissionWindowTask < Task
 
   before_validation :set_assignee
 
+  # also accounts for when EvidenceSubmissionWindownTask is manually closed by the user
```

Comment:
```suggestion
  # also called when EvidenceSubmissionWindownTask is manually closed by the user
```

I'm not certain about this wording. I initially took this comment as meaning that the `when_timer_ends` method was considering an EvidenceSubmissionWindowTask when you hadn't modified it and I was confused. Saying it's also called is clearer to me, but only go with this if you agree that it's clearer.